### PR TITLE
Added task type filter on contributor dashboard.

### PR DESCRIPTION
**Pull request by shivam-r**

### CODE REVIEW

File: DCode.Web/Views/Contributor/_ContributorTaskApplication.cshtml [hunk 1]
```diff
@@ -199,10 +204,14 @@
             </div>
             <div class="modal-body text-center" style="padding-bottom:120px;">
                 <h3 class="dark-grey-blue font-size-24">Congratulations!</h3>
-                <div class="gray-text font-size-16 opacity-normal">
+                <div class="gray-text font-size-16 opacity-normal" ng-show="$scope.taskSearch.selectedTaskType == 1">
```
Comment (ritwiksen) [inline DCode.Web/Views/Contributor/_ContributorTaskApplication.cshtml:207]: $scope would not be necessary in cshtml

File: DCode.Web/Views/Contributor/_ContributorTaskApplication.cshtml [hunk 2]
```diff
@@ -199,10 +204,14 @@
             </div>
             <div class="modal-body text-center" style="padding-bottom:120px;">
                 <h3 class="dark-grey-blue font-size-24">Congratulations!</h3>
-                <div class="gray-text font-size-16 opacity-normal">
+                <div class="gray-text font-size-16 opacity-normal" ng-show="$scope.taskSearch.selectedTaskType == 1">
                     <div>You have requested permission for <span class="color-8e44ad">{{taskRequest.TaskName}}</span> under project <span class="color-8e44ad">{{taskRequest.ProjectName}}</span> for {{taskRequest.Hours}} hrs starting {{taskRequest.StartingDate}}.</div>
                     <div>While you wait for your manager to get back, complete your <span class="color-8e44ad link-text" ng-click="NavigateToProfile()">Profile</span> for better task recommendations.</div>
                 </div>
+                <div class="gray-text font-size-16 opacity-normal" ng-show="$scope.taskSearch.selectedTaskType == 2">
```
Comment (ritwiksen) [inline DCode.Web/Views/Contributor/_ContributorTaskApplication.cshtml:211]: $scope would not be necessary in cshtml

**Review by ritwiksen — COMMENTED** [review]
Minor alterations needed

File: DCode.Web/Views/Contributor/_ContributorTaskApplication.cshtml (hunk 1 above)
Comment (shivam-r) [reply DCode.Web/Views/Contributor/_ContributorTaskApplication.cshtml:207]: Agree. Fixed in my second PR.

File: DCode.Web/Views/Contributor/_ContributorTaskApplication.cshtml (hunk 2 above)
Comment (shivam-r) [reply DCode.Web/Views/Contributor/_ContributorTaskApplication.cshtml:211]: Fixed in second PR.

**Review by ritwiksen — APPROVED** [review]

**Pull request merged**